ui/pages/onboarding-flow/metametrics/metametrics.js

```diff
@@ -94,40 +102,62 @@ export default function OnboardingMetametrics() {
           {t('onboardingMetametricsSendAnonymize')}
         </li>
         <li>
-          <i className="fa fa-times" />
-          {t('onboardingMetametricsNeverCollect', [
-            <Typography
-              variant={TypographyVariant.Span}
-              key="never"
-              fontWeight={FONT_WEIGHT.BOLD}
-            >
-              {t('onboardingMetametricsNeverEmphasis')}
-            </Typography>,
-          ])}
+          <Box>
+            <Icon
+              marginInlineEnd={2}
+              name={ICON_NAMES.CLOSE}
+              size={ICON_SIZES.SM}
+              color={Color.errorDefault}
+            />
+            {t('onboardingMetametricsNeverCollect', [
+              <Typography
+                variant={TypographyVariant.span}
+                key="never"
+                fontWeight={FONT_WEIGHT.BOLD}
+                marginTop={0}
+              >
+                {t('onboardingMetametricsNeverEmphasis')}
+              </Typography>,
+            ])}
+          </Box>
         </li>
         <li>
-          <i className="fa fa-times" />
-          {t('onboardingMetametricsNeverCollectIP', [
-            <Typography
-              variant={TypographyVariant.Span}
-              key="never-collect"
-              fontWeight={FONT_WEIGHT.BOLD}
-            >
-              {t('onboardingMetametricsNeverEmphasis')}
-            </Typography>,
-          ])}
+          <Box>
+            <Icon
+              marginInlineEnd={2}
+              name={ICON_NAMES.CLOSE}
+              size={ICON_SIZES.SM}
+              color={Color.errorDefault}
+            />
+            {t('onboardingMetametricsNeverCollectIP', [
+              <Typography
+                variant={TypographyVariant.span}
+                key="never-collect"
+                fontWeight={FONT_WEIGHT.BOLD}
+              >
+                {t('onboardingMetametricsNeverEmphasis')}
+              </Typography>,
+            ])}
+          </Box>
         </li>
         <li>
-          <i className="fa fa-times" />
-          {t('onboardingMetametricsNeverSellData', [
-            <Typography
-              variant={TypographyVariant.Span}
-              key="never-sell"
-              fontWeight={FONT_WEIGHT.BOLD}
-            >
-              {t('onboardingMetametricsNeverEmphasis')}
-            </Typography>,
-          ])}
+          <Box>
+            <Icon
+              marginInlineEnd={2}
+              name={ICON_NAMES.CLOSE}
+              size={ICON_SIZES.SM}
+              color={Color.errorDefault}
+            />
+            {t('onboardingMetametricsNeverSellData', [
+              <Typography
+                variant={TypographyVariant.span}
+                key="never-sell"
+                fontWeight={FONT_WEIGHT.BOLD}
+              >
+                {t('onboardingMetametricsNeverEmphasis')}
+              </Typography>,
+            ])}
+          </Box>{' '}
         </li>
       </ul>
       <Typography
```

Comment:
Would be good to get a screenshot

Reply:
Screenshot is in the PR description.

Reply:
<img width="374" alt="220457880-87926139-f4e7-4780-ad6f-46be5e873d99" src="https://user-images.githubusercontent.com/46655/220728565-54bdf0c9-987b-41d0-bd19-f2b92b3edb43.png">

Would love advice on how to perfectly center this.  There's a storybook for quick view.